Yank/repex.py

```diff
@@ -1899,48 +1858,55 @@ def is_completed(self):
     # Main public interface.
     # -------------------------------------------------------------------------
 
-    def create(self, thermodynamic_states, sampler_states, storage, unsampled_thermodynamic_states=None, metadata=None):
-        """Create new replica-exchange simulation.
+    title_template = ('Multi-state sampler simulation created using MultiStateSampler class '
+                      'of yank.repex on {}')
+
+    def create(self, thermodynamic_states, sampler_states, storage,
```

Comment:
I actually just had a thought for the YAML hooks and generalization. I don't like this call because the `ReplicaExchange` uses a _different_ call method, which means there is no way to generalize this for things like `yank.AlchemicalPhase`

Reply:
Specifically its the `initial_thermodynamic_states` mandatory arg which would need to be adapted. I think we can cast it to a keyword argument with default behavior. I already kind of did this for the test suite with a line like: 

```python
init_thermo_states = np.tile(range(n_states), int(np.ceil(n_sampler/n_states)))[:n_sampler]
```